plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/pom.xml

```diff
@@ -11,56 +11,4 @@
   <artifactId>com.google.cloud.tools.eclipse.appengine.standard.java8</artifactId>
   <version>0.1.0-SNAPSHOT</version>
   <packaging>eclipse-plugin</packaging>
-
-  <dependencies>
```

Comment:
It's nearly done.  It was done, but there are now conflicts and kokoro seems to be failing.  Will revisit it soon.